### ISSUE TITLE
Some UI components are not reachable using Talkback or Keyboard

### ISSUE THREAD

**Issue by dessalines** [LemmyNet/jerboa#1505]
### Pre-Flight checklist

- [X] Did you check to see if this issue already exists?
- [X] This is a single feature request. (Do not put multiple feature requests in one issue)
- [X] This is not a question or discussion. (Use https://lemmy.ml/c/jerboa for that)

### Describe The Feature Request Below

Several UI components are not reachable using Talkback or keyboard. 

For example the clear button (X) on the right side of the search box. And the list menu for ‘instance’ under ‘Add account’>Login must be clicked again when activated, but this is not possible with Talkback or using a keyboard.
